drivers/copy/copy_linux.go

```diff
@@ -271,6 +272,19 @@ func doCopyXattrs(srcPath, dstPath string) error {
 		return err
 	}
 
+	xattrs, err := system.Llistxattr(srcPath)
+	if err != nil && err != unix.EOPNOTSUPP {
```

Comment:
What happens if err == unix.EOPNOTSUPP?  Wouldn't the calls to copyXattr below fail?

Reply:
In that case xattrs == nil, so it never enters the for loop.

Reply:
Ah ok.